### DISCUSSION TITLE
Proxy Hosts "Reference Note" for each host created

### DISCUSSION THREAD

**Discussion by admecoach** [NginxProxyManager/nginx-proxy-manager#2328] [Ideas]
One thing that I feel would help many people is a simple note field on the proxy hosts page under each proxy host.  Or even call it a comment line. Between having a simple reference to what the intent was for the proxy in the first place and maybe even a ping to check advise if an actual domain dns record is still in place for the proxied location, I think developers would find this helpful.  I try to keep a cross reference in Notion for all these to have further notes but with so many (I guess this only apples to people with dozens of proxy hosts like me) this would be helpful to reference what project the sub domain or domain was for when setup.

Suggested Location of Note Field - Under the Destination Col below the IP, similar to how the Created Date exists under the Source Col with an editable one line reading:  'Note: Private reference note about this proxy host'

This idea probably pairs with the category enhancement suggestion:  #1314  / [categories and tags](https://github.com/NginxProxyManager/nginx-proxy-manager/discussions/1314)

![npm-note-suggestion](https://user-images.githubusercontent.com/5846373/196089164-17843d3a-3241-4feb-9f4f-1f9b659de507.jpg)

**Comment by popliviustefan** [comment]
This would be a great quality of life addition! 
And the note could be added in one of multiple ways: 
- as second line, as you suggested
- as a separate column after Status
- As a hover pop-up
